KioskTests/Bid Fulfillment/KeypadViewModelTests.swift

```diff
@@ -6,109 +6,120 @@ import Kiosk
 
 class KeypadViewModelTestClass: NSObject {
     // Start with invalid data
-    dynamic var stringValue = "something invalid"
-    dynamic var intValue = -1
+    var stringValue = Variable("something invalid")
+    var intValue = Variable(-1)
 }
 
 class KeypadViewModelTests: QuickSpec {
     override func spec() {
         var subject: KeypadViewModel!
         var testHarness: KeypadViewModelTestClass!
+        var disposeBag: DisposeBag!
         
         beforeEach {
             subject = KeypadViewModel()
             testHarness = KeypadViewModelTestClass()
+            disposeBag = DisposeBag()
+
+            subject
+                .stringValue
+                .bindTo(testHarness.stringValue)
+                .addDisposableTo(disposeBag)
+            subject
+                .intValue
+                .bindTo(testHarness.intValue)
+                .addDisposableTo(disposeBag)
         }
         
         it("it has default values") {
-            RAC(testHarness, "stringValue") <~ subject.stringValueSignal
-            RAC(testHarness, "intValue") <~ subject.intValueSignal
-            
             expect(testHarness.intValue) == 0
             expect(testHarness.stringValue) == ""
         }
         
         it("adds digits") {
-            waitUntil { (done) -> Void in
-                RAC(testHarness, "stringValue") <~ subject.stringValueSignal
-                RAC(testHarness, "intValue") <~ subject.intValueSignal
-                
-                [1,3,3,7].reduce(RACSignal.empty(), combine: { (signal, input) -> RACSignal in
-                    signal.then { subject.addDigitCommand.execute(input) }
-                }).subscribeCompleted { () -> Void in
-                    expect(testHarness.intValue) == 1337
-                    expect(testHarness.stringValue) == "1337"
-                    
-                    done()
-                }
+            waitUntil { done in
```

Comment:
There is none, strictly. It's just a pattern I applied from elsewhere ("wait until" the signal has completed). I can remove it, but I kind of like it.